docs/subnets/register-validate-mine.md

```diff
@@ -29,20 +38,20 @@ Registering works like this:
 :::tip Ownership belongs to a hotkey
 Ownership belongs to a hotkey. Hence, for example, when you delegate your TAO to a subnet validator, you attach your delegated TAO to the hotkey of the subnet validator. See [Delegation](../staking-and-delegation/delegation.md). 
 
-A hotkey can hold multiple UIDs across **separate** subnets. However within one subnet, each UID must have a unique hotkey. 
+A hotkey can hold multiple UIDs across **separate** subnets. However, within one subnet, each UID must have a unique hotkey.
 :::
 
-After you register your keys, you can then proceed to become either a subnet miner or a subnet validator, which have different requirements. See below. 
+After registering your keys, you can become either a subnet miner or a subnet validator with different requirements. See below.
```

Comment:
I think maybe the first sentence reads better this way: "After registering your keys, you can become either a subnet miner or a subnet validator. Note that they have different requirements."